custom_components/hikvision_axpro/__init__.py

```diff
@@ -474,3 +524,37 @@ async def relay_off(self, relay_id: int):
             self._relay_call, relay_id, False
         )
         return response.status_code == 1
+
+    def _siren_call(self, relay_id: int, is_enabled: bool) -> JSONResponseStatus:
+        api_url = "/ISAPI/SecurityCP/control/siren/{}"
```

Comment:
@petrleocompel this endpoint should probably added to core library

Reply:
Yup, after testing in beta I normally merge it to core lib and then publish with new version.